pkgs/applications/misc/mangal/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, stdenv, buildGoModule, go_1_18, fetchFromGitHub, installShellFiles }:
+
+buildGoModule.override ({ go = go_1_18; }) rec {
```

There is no need to override go_1_18, since is the default version used in nixpkgs
```suggestion
{ lib, stdenv, buildGoModule, fetchFromGitHub, installShellFiles }:

buildGoModule rec {
```
besides that, LGTM

---

The default go version is 1.17 and some of the libraries use features that are only supported by 1.18+ . That's why I'm using the override

---

Never mind, my branch was too many commits behind and was still using 1.17 as default. Your suggestion is correct.